src/drive/web/modules/drive/SharingsContainer.jsx

```diff
@@ -25,13 +25,23 @@ class SharingFetcher extends React.Component {
       const resp = await client
         .collection('io.cozy.files')
         .all({ keys: sharedDocuments })
+
+      const parent_ids = resp.data.map(f => f.dir_id)
+      const parentsResp = await client.collection('io.cozy.files').all({
+        keys: parent_ids
+      })
+      const parents = parentsResp.data
```

Comment:
We only fetch the first parent, right? 

So if my file is in : 
/Admin/foo/bar/toto.PNG

we'll only display : 
/bar/

right?

Reply:
OK just read the test... Sorry.

Reply:
The path we get from the stack is the full path of the directory